contracts/test/AllowanceTarget/Constructor.t.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.7.6;
+
+import "contracts-test/AllowanceTarget/Setup.t.sol";
+
+contract TestAllowanceTargetConstructor is TestAllowanceTarget {
+    function testCannotDeployByZeroAddress() public {
```

Comment:
I think we need to differentiate between test case names event if the test cases are separated in different files. For example `testCannotDeployByZeroAddress` and `testConstructor` could exist in most of the contracts' test cases, so if you do `forge test --match-test testConstructor` it will run all `testConstructor ` test cases in those contracts' test cases.

Reply:
But this way the case will still have lengthy naming. This is a workaround so not the most ideal way and I'm ok with sacrificing `--match-test` for short case naming.

Reply:
If the main goal is to reduce lines in a single file, not the length of the name, then it make sense.

Reply:
I think this is not a problem since we can use `--match-test` together with `--match-contract` or `--match-path`. 

I think lengthy case name is one of the pain points we try to solve by slice tests into multiple files, right?

Reply:
Thanks for suggestions! Maybe we should discuss the choice of long and short test func names.